Xamarin.Forms.Platform.UAP/ImageButtonRenderer.cs

```diff
@@ -18,10 +18,24 @@ public class ImageButtonRenderer : ViewRenderer<ImageButton, FormsButton>, IImag
 		bool _disposed;
 		WImage _image;
 		FormsButton _formsButton;
+		Window _window;
 
 		public ImageButtonRenderer() : base()
 		{
 			ImageElementManager.Init(this);
+			_window = Window.Current;
+			_window.Activated += Window_Activated;
```

Comment:
Do we need to unsubscribe this somewhere?

Reply:
Yes will do.